services/compute/virtualmachine/virtualmachine.go

```diff
@@ -420,6 +440,8 @@ func (c *client) getVirtualMachine(vm *wssdcloudcompute.VirtualMachine, group st
 			SecurityProfile:         c.getVirtualMachineSecurityProfile(vm),
 			OsProfile:               c.getVirtualMachineOSProfile(vm.Os),
 			NetworkProfile:          c.getVirtualMachineNetworkProfile(vm.Network),
+			GuestAgentProfile:       c.getVirtualMachineGuestAgentProfile(vm.GuestAgent),
```

Comment:
This would be impacted without the cloudagent code? as all vm would try to read this info